Sources/NIOCore/Docs.docc/swift-concurrency.md

```diff
@@ -0,0 +1,244 @@
+# NIO and Swift Concurrency
+
+This article explains how to interface between NIO and Swift Concurrency 
+
+NIO was created before native Concurrency support in Swift existed, hence, NIO had to solve
+a few problems that have solutions in the language today. Since, the introduction of Swift Concurrency
+NIO has added numerous features to make the inter-op between NIOs channel eventing system and Swift's
+Concurrency primitives as easy as possible.
+
+### EventLoopFuture bridges
+
+The first bridges, that NIO introduced were adding methods on ``EventLoopFuture`` and ``EventLoopPromise``
+to bridge to and from Concurrency. Namely these bridges are ``EventLoopFuture/get()`` and ``EventLoopPromise/completeWithTask(_:)``.
+
+> Warning: The future ``EventLoopFuture/get()`` method does not support task cancellation.
```

Comment:
```suggestion
> Warning: ``EventLoopFuture/get()`` does not support task cancellation.
```

Reply:
I did add the words around because docc renders this as just `get()` which reads very weird.

Reply:
Ah I see. Okay, then it's fine - with the word `future` though, did you mean `ELF`?